DICOM/src/Microsoft.Health.Dicom.Anonymizer.CommanLineTool.UnitTests/AnonymizerToolUnitTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.IO;
+using System.Threading.Tasks;
+using Dicom;
+using Microsoft.Health.Dicom.Anonymizer.Core.Tool;
+using Xunit;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.UnitTests
+{
+    public class AnonymizerToolUnitTests
```

Comment:
>AnonymizerToolUnitTests [](http://example.com/codeflow?start=17&length=23)

you are missing the tests that have argument issues so you can test your logic to see if it works as you expect.

Reply:
I will add more tests next pr.